cube/src/main/resources/META-INF/services/org.jboss.arquillian.core.spi.LoadableExtension

```diff
@@ -0,0 +1 @@
+org.jboss.arquillian.ce.cube.CECubeOpenShiftExtension
```

Comment:
Personal preference, to a degree.  Manual method reduces the number of external dependencies required to build the project.